src/Examine.Lucene/Suggest/DirectSpellCheckerDefinition.cs

```diff
@@ -0,0 +1,48 @@
+using Lucene.Net.Index;
+using Lucene.Net.Search.Spell;
+using System.Linq;
+using Examine.Suggest;
+
+namespace Examine.Lucene.Suggest
+{
+    public class DirectSpellCheckerDefinition : LuceneSuggesterDefinition, ILookupExecutor
+    {
+        public DirectSpellCheckerDefinition(string name, string[] sourceFields = null, ISuggesterDirectoryFactory directoryFactory = null)
+            : base(name, sourceFields, directoryFactory)
+        {
+        }
+        DirectSpellChecker Spellchecker { get; set; }
+
+        /// <inheritdoc/>
+        public override ILookupExecutor BuildSuggester(FieldValueTypeCollection fieldValueTypeCollection, ReaderManager readerManager, bool rebuild)
+            => BuildDirectSpellCheckerSuggester(fieldValueTypeCollection, readerManager, rebuild);
+
+        /// <inheritdoc/>
+        public override ISuggestionResults ExecuteSuggester(string searchText, ISuggestionExecutionContext suggestionExecutionContext) => ExecuteDirectSpellChecker(searchText, suggestionExecutionContext);
+        protected ILookupExecutor BuildDirectSpellCheckerSuggester(FieldValueTypeCollection fieldValueTypeCollection, ReaderManager readerManager, bool rebuild)
+        {
+            Spellchecker = new DirectSpellChecker();
+            return this;
+        }
+
+        private ISuggestionResults ExecuteDirectSpellChecker(string searchText, ISuggestionExecutionContext suggestionExecutionContext)
+        {
+            string field = SourceFields.First();
```

Comment:
Same here what happens if `sourceFields` are null or empty

Reply:
Added checks to constructor and inline. Changed constructor signature to remove null defaults.